src/main/java/appeng/parts/misc/ItemHandlerAdapter.java

```diff
@@ -167,7 +172,14 @@ public IAEItemStack extractItems( IAEItemStack request, Actionable mode, IAction
 		{
 			if( mode == Actionable.MODULATE )
 			{
-				this.onTick();
+				try
+				{
+					this.proxyable.getProxy().getTick().alertDevice( this.proxyable.getProxy().getNode() );
+				}
+				catch( GridAccessException ex )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1166)